django/contrib/auth/hashers.py

```diff
@@ -570,7 +570,7 @@ class ScryptPasswordHasher(BasePasswordHasher):
     algorithm = "scrypt"
     block_size = 8
     maxmem = 0
-    parallelism = 1
+    parallelism = 5
```

Comment:
This change should be mentioned in release notes.

Reply:
I'll add a release note, thank you for your review :)